types/mongodb/index.d.ts

```diff
@@ -30,7 +30,7 @@ export { Binary, Double, Long, Decimal128, MaxKey, MinKey, ObjectID, ObjectId, T
 
 // Class documentation : http://mongodb.github.io/node-mongodb-native/3.0/api/MongoClient.html
 export class MongoClient extends EventEmitter {
-    constructor();
+    constructor(uri?: string, options?: MongoClientOptions);
```

Comment:
From the documentation, the URL argument is required

Reply:
ok, I will update it, looks like you're right